packages/dds/merge-tree/src/test/mergeTreeOperationRunner.ts

```diff
@@ -37,7 +38,25 @@ export const obliterateRange: TestOperation = (
 	client: TestClient,
 	opStart: number,
 	opEnd: number,
-) => client.obliterateRangeLocal(opStart, opEnd);
```

Comment:
can we keep number based obliterate mixed in, something like `if(random.bool()){//do number}else{//do sided}`